usaspending_api/agencies/management/commands/load_agencies.py

```diff
@@ -0,0 +1,276 @@
+    ProcessingStep("Create toptier agency temp table", "toptier_agency_create_temp_table", None, False, False),
+    ProcessingStep("Delete obsolete toptier agencies", "toptier_agency_delete", None, True, False),
+    ProcessingStep("Update existing toptier agencies", "toptier_agency_update", None, True, False),
+    ProcessingStep("Create new toptier agencies", "toptier_agency_create", None, True, False),
+    ProcessingStep("Create subtier agency temp table", "subtier_agency_create_temp_table", None, False, False),
+    ProcessingStep("Delete obsolete subtier agencies", "subtier_agency_delete", None, True, False),
+    ProcessingStep("Update existing subtier agencies", "subtier_agency_update", None, True, False),
+    ProcessingStep("Create new subtier agencies", "subtier_agency_create", None, True, False),
+    ProcessingStep("Create agency temp table", "agency_create_temp_table", None, False, False),
+    ProcessingStep("Delete obsolete agencies", "agency_delete", None, True, False),
+    ProcessingStep("Update existing agencies", "agency_update", None, True, False),
+    ProcessingStep("Create new agencies", "agency_create", None, True, False),
+    ProcessingStep(
+        "Update treasury appropriation accounts", "treasury_appropriation_account_update", None, False, True
+    ),
+    ProcessingStep("Update transactions", "transaction_normalized_update", None, False, True),
+    ProcessingStep("Update awards", "award_update", None, False, True),
+    ProcessingStep("Update subawards", "subaward_update", None, False, True),
+    ProcessingStep("Clean up", "clean_up", None, False, False),
+]
+
+SQL_PATH = Path(__file__).resolve().parent / "load_agencies_sql"
+
+
+class Command(BaseCommand):
+
+    help = (
+        "Loads CGACs, FRECs, Subtier Agencies, Toptier Agencies, and Agencies.  Load is all or nothing.  "
+        "If anything fails, nothing gets saved."
+    )
+
+    agency_file = None
+    connection = get_connection(read_only=False)
+    force = False
+    rows_affected_count = 0
+
+    def add_arguments(self, parser):
+
+        parser.add_argument(
+            "agency_file",
+            metavar="AGENCY_FILE",
+            help="Path (for local files) or URI (for http(s) or S3 files) of the raw agency CSV file to be loaded.",
+        )
+
+        parser.add_argument(
+            "--force",
+            action="store_true",
+            help=(
+                "Reloads agencies even if the change max change threshold of {:,} is exceeded.  This is a safety "
+                "feature to prevent accidentally updating every award, transaction, and subaward in the system as "
+                "part of the nightly pipeline.".format(MAX_CHANGES)
+            ),
+        )
+
+    def handle(self, *args, **options):
+        self.agency_file = options["agency_file"]
+        logger.info("AGENCY FILE: {}".format(self.agency_file))
+
+        self.force = options["force"]
+        logger.info("FORCE: {}".format(self.force))
```

Comment:
```suggestion
        logger.info("CHANGE LIMIT: {}".format(str(MAX_CHANGES) if self.force else "unlimited"))
```
it will be easy to miss what "FORCE" is in the future, and accidentally putting in too many agencies for one night might be a source of confusion in the future.